src/codewhisperer/commands/invokeRecommendation.ts

```diff
@@ -61,10 +61,14 @@ export async function invokeRecommendation(
         if (RecommendationHandler.instance.isGenerateRecommendationInProgress) {
             return
         }
+        RecommendationHandler.instance.checkAndResetCancellationTokens()
```

Comment:
Can you explain why this fixed the issue and how you did the troubleshooting

Reply:
I synced with Andrew, we suspected that it returned early [here](https://github.com/aws/aws-toolkit-vscode/blob/master/src/codewhisperer/service/recommendationHandler.ts#L311) as we were able to see the logs before this, and verified the fix in C9

Reply:
Thanks. This is useful info to include in the commit message (and PR description):

> we suspected that it returned early [here](https://github.com/aws/aws-toolkit-vscode/blob/master/src/codewhisperer/service/recommendationHandler.ts#L311)

This helps in the future when others need to debug these issues. I was pretty lost when I was debugging this :)